pkg/kubelet/kuberuntime/kuberuntime_container_test.go

```diff
@@ -133,6 +134,7 @@ func TestKillContainer(t *testing.T) {
 // the internal type (i.e., toKubeContainerStatus()) for containers in
 // different states.
 func TestToKubeContainerStatus(t *testing.T) {
+	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.InPlacePodVerticalScaling, true)()
```

Comment:
This now only will run our existing unit tests with the feature gate on, we also need to test with the feature gate off and ensure existing behaviour is preserved.

Reply:
Good catch! Will be fixed in next update.

Reply:
Preserved existing test and added new test for feature enabled.
https://github.com/kubernetes/kubernetes/pull/102884/files#diff-5365de1136faccf8b1f9232277143c56c16b67e4d02b225eefae265c07d1c3c5R236